### ISSUE TITLE
Replace multi-user.target with default.target in systemd user unit

### ISSUE THREAD

**Issue by Frederick888** [containers/libpod#4294]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**
Currently Podman generates only one `io.podman.service` which is used downstream for both systemd system and user units. However `multi-user.target` does not exist for user units.

**Steps to reproduce the issue:**

1. Install podman ([Arch Linux package](https://www.archlinux.org/packages/community/x86_64/podman/) in my case)
2. `systemctl --user enable io.podman.service`

**Describe the results you received:**
```sh
$ systemctl --user enable io.podman.service
Failed to enable unit: No such file or directory
```

**Describe the results you expected:**
User unit can be enabled.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `podman version`:**

```
Version:            1.6.1
RemoteAPI Version:  1
Go Version:         go1.13.1
OS/Arch:            linux/amd64
```

**Output of `podman info --debug`:**

```
debug:
  compiler: gc
  git commit: ""
  go version: go1.13.1
  podman version: 1.6.1
host:
  BuildahVersion: 1.11.2
  CgroupVersion: v1
  Conmon:
    package: Unknown
    path: /usr/bin/conmon
    version: 'conmon version 2.0.1, commit: 4dc8bcfec41e10ca760c8e2089474c2843dfd066'
  Distribution:
    distribution: arch
    version: unknown
  MemFree: 18826465280
  MemTotal: 33635438592
  OCIRuntime:
    package: Unknown
    path: /usr/bin/runc
    version: |-
      runc version 1.0.0-rc9
      commit: d736ef14f0288d6993a1845745d6756cfc9ddd5a
      spec: 1.0.1-dev
  SwapFree: 7991980032
  SwapTotal: 7999582208
  arch: amd64
  cpus: 8
  eventlogger: journald
  hostname: FredArch
  kernel: 5.3.6-arch1-1-ARCH
  os: linux
  rootless: true
  slirp4netns:
    Executable: /bin/slirp4netns
    Package: Unknown
    Version: |-
      slirp4netns version 0.4.1
      commit: 4d38845e2e311b684fc8d1c775c725bfcd5ddc27
  uptime: 2h 10m 10.01s (Approximately 0.08 days)
registries:
  blocked: null
  insecure: null
  search:
  - docker.io
  - registry.fedoraproject.org
  - quay.io
  - registry.access.redhat.com
  - registry.centos.org
store:
  ConfigFile: /home/frederick/.config/containers/storage.conf
  ContainerStore:
    number: 0
  GraphDriverName: overlay
  GraphOptions:
    overlay.mount_program:
      Executable: /bin/fuse-overlayfs
      Package: Unknown
      Version: |-
        fusermount3 version: 3.7.0
        fuse-overlayfs: version 0.6.4
        FUSE library version 3.7.0
        using FUSE kernel interface version 7.31
  GraphRoot: /home/frederick/.local/share/containers/storage
  GraphStatus:
    Backing Filesystem: extfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  ImageStore:
    number: 0
  RunRoot: /run/user/1000
  VolumePath: /home/frederick/.local/share/containers/storage/volumes
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
$ yay -Q podman
podman 1.6.1-1
```

**Comment by vrothberg** [comment]
I think we need two different files for system and user. The upstream file targets only the system case and I believe that multi-user target is needed to make sure all relevant system services are up.

Shall we provide another file? @edsantiago @rhatdan @mheon @baude WDYT?

**Comment by cgwalters** [comment]
Isn't this fixed by https://github.com/containers/libpod/pull/4231 ?

**Comment by Frederick888** [comment]
Ah sorry, seems that it's been fixed indeed. I'll close this issue now.

**Comment by vrothberg** [comment]
> Isn't this fixed by #4231 ?

Good catch! I didn't check the Makefile :)
